src/math/matrix.c

```diff
@@ -31,7 +31,7 @@ int mat_multiply(struct mat_matrix_16b *a, struct mat_matrix_16b *b, struct mat_
 	if (a->columns != b->rows || a->rows != c->rows || b->columns != c->columns)
 		return -EINVAL;
 
-	int64_t s;
+	int32_t s;  /* Changed from int64_t to int32_t */
```

Comment:
this is a comment in the code. Nobody will see the history when reading it. A "changed" comment implies that before it was something different, then someone changed it. But when you read code, you aren't that interested in the history, you want to know how and why it works in its present version. So comments should describe the present code, not its history.